solaris/solaris11/README.md

```diff
@@ -0,0 +1,68 @@
+Wazuh Solaris11 packages
+==================
+
+In this repository, you can find the necessary tools to build a Wazuh package for Solaris 11.
+
+## Tools needed to build the package
+
+To build a Wazuh package you need to install the following tools:
+  - `git`: `pkgutil -i git`.
+
+## Building Solaris11 packages
+
+To build a Solaris 10 package, you need to download this repository and use the `generate_wazuh_packages.sh` script. This script will download the source code from the [wazuh/wazuh](https://github.com/wazuh/wazuh) repository and generate a `.p5p` package.
```

Comment:
Solaris 10 or 11?

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/191/commits/49424d4fb30ffd2950a5e8cb0d1fac8c69fdf893